docs/dev-contributing.md

```diff
@@ -42,6 +49,8 @@ Reporting bugs, feature requests, and questions that you have about the platform
 
 * If you are filing a bug, please include information about the operating system, device, and browser that you were using when you saw the bug. _Example: Seen on Windows 10 with Firefox 66.0.5 on Oculus Rift_
 
+* A good format is a sequence of steps, ending with the expected result, and the actual result.
```

Comment:
This should probably be stronger, and I think the expected/actual should be switched (at least that is what is suggested in the bug report templates for the Hubs Client and Hubs Blender add-on).  E.g.
```suggestion
* Make sure to include the sequence of steps to reproduce the bug, ending with the actual result, and then describe what you expected to happen instead in the next section.
```